composer.json

```diff
@@ -28,9 +28,9 @@
         "identity"
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": "^8.1",
         "ext-json": "*",
-        "facile-it/php-jose-verifier": "^0.3 || ^0.4.3",
+        "facile-it/php-jose-verifier": "^0.5.0-beta1",
```

Comment:
TODO: if everything is ok, we should re-tag as stable and change the requirement here.

Reply:
Is the 0.5 tag available on that repo yet? I've an issue open there but it is still open?

Reply:
Not yet, I'm thinking that we can use the beta version, and tag this PR in beta too